README.md

```diff
@@ -343,5 +352,33 @@ The generated report will be available at ```target/site/jacoco/index.html```
+
+## Configuration options
+
+The `UnleashConfig` class exposes a set of builder methods to configure your Unleash client. The available options are listed below with a description of what they do. For the full signatures, take a look at the [`UnleashConfig` class definition](src/main/java/io/getunleash/util/UnleashConfig.java).
+
+| Method name                                | Description                                                                                                                                                         | Required | Default value                        |
+|--------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|--------------------------------------|
+| `appName`                                  | The name of the application as shown in the Unleash UI. Registered applications are listed on the Applications page.                                                | Yes      | `null`                               |
+| `backupFile`                               | The path to the file where [local backups](#local-backup) get stored.                                                                                               | No       | `"java.io.tmpdir/unleash-repo.json"` |
+| `customHttpHeader`                         | Add a [custom HTTP header](#custom-http-headers) to the list of HTTP headers that will the client sends to the Unleash API. Each method call will add a new header. | No       | N/A                                  |
+| `customHttpHeadersProvider`                | Add a custom HTTP header provider. Useful for [dynamic custom HTTP headers](#dynamic-custom-http-headers).                                                          | No       | `null`                               |
+| `disableMetrics`                           | A boolean indicating whether the client should disable sending usage metrics to the Unleash server.                                                                 | No       | `false`                              |
+| `enableProxyAuthenticationByJvmProperties` | Enable support for [using JVM properties for HTTP proxy authentication](#http-proxy-with-authentication).                                                           | No       | `false`                              |
+| `environment`                              | The name of the current environment.                                                                                                                                | No       | `null`                               |
+| `fallbackStrategy`                         | ???                                                                                                                                                                 | No       | `null`                               |
```

Comment:
What is this? Fallback strategy for what?

Reply:
Looks like this is used when checking the strategies for a toggle. When the SDK tries to lookup the strategy for a given toggle it uses this as a fallback if the strategy can't be mapped to one of the SDK strategy types (guessing this is for a usecase where you may be running an older SDK with a newer Unleash with new strategies)